client/src/components/TrailsPage.js

```diff
@@ -57,19 +59,30 @@ const TrailsPage = (props) => {
   }
   const trailListItems = trails.map((trailItem) => {
     return (
-      <li key={trailItem.id}>
-        <Link to={`/trails/${trailItem.id}`}>{trailItem.name}</Link>
-      </li>
+      <div>
+        <li key={trailItem.id}>
+          <Link to={`/trails/${trailItem.id}`}>{trailItem.name}</Link>
+        </li>
+      </div>
```

Comment:
You should also see a warning in your browser console from this because the top-level element (`div`) does not have the key, it's attached to the `li`. Technically don't need the surrounding `div` since the `li` is the top-most element being returned.

Reply:
OK, I see, thank you for the heads up about the key!